### DISCUSSION TITLE
Sending a value from POS into CRT

### DISCUSSION THREAD

**Discussion by Danialddm** [microsoft/Dynamics365Commerce.Solutions#338] [Q&A]
Hi experts,
How can I send a string value from POS side into CRT?
I have created a separate request and response in both sides but when i want to capture the value, it is null.
Really appreciate any urgent help... 
Thanks 
Regards

**Comment by dsdharmen91** [comment]
Hi we can do it using customizations. you can reach to dsdharmen91@gmail.com
